Reword doc v2202 (#83)
NVIDIA/spark-rapids-examples: examples/Spark-ETL+XGBoost/README.md

# Spark-ETL+XGBoost examples

## 1. Prepare the dataset Spark-ETL+XGBoost examples codes

You can get a small size datasets for each example in the [datasets](../../datasets) folder. 
These datasets are only provided for convenience. In order to test for performance, 
please prepare packages and download a larger dataset by following links:
- Prepare packages and datasets
    - [Scala](../../docs/get-started/xgboost-examples/prepare-package-data/preparation-scala.md)
    - [Python](../../docs/get-started/xgboost-examples/prepare-package-data/preparation-python.md)

We provided a [Morgage Dataset (1 GB uncompressed)](https://rapidsai-data.s3.us-east-2.amazonaws.com/spark/mortgage.zip), 

> which is used in the Mortgage demo blow for quick download.

Reviewer: ```suggestion
which is used in the Mortgage demo below for quick download.
```

> which is used in the Mortgage demo blow for quick download.

Reviewer 2: ```suggestion
which is used in the Mortgage demo below for quick download.
```

Author: this should already be updated in the latest commits.

## 2. Spark-ETL+XGBoost getting started guides

> We have three sub demos to demonstrate the Spark ETL + XGBoost process as Mortgage, Taxi and Agaricus(Agaricus demo has 

Reviewer: ```suggestion
We have three examples to demonstrate the Spark ETL + XGBoost process: Mortgage, Taxi and Agaricus(Agaricus demo has 
```

very simple ETL part as just a filter process, so we combined the ETL part into XGBoost process).

> You have two ways to get start, you can start by running scala or python codes, or you can start by running notebooks.

Reviewer: ```suggestion
Here are 2 ways to get started:
- Running Spark jobs
- Notebooks
```

> ### 1. Getting Started From Codes

Reviewer: ```suggestion
### 1. Getting Started by running Spark jobs
```

- Taxi: [Scala](../../examples/Spark-ETL+XGBoost/taxi/scala/src/com/nvidia/spark/examples/taxi), [Python](../../examples/Spark-ETL+XGBoost/taxi/python/com/nvidia/spark/examples/taxi)
- Agaricus: [Scala](../../examples/Spark-ETL+XGBoost/agaricus/scala/src/com/nvidia/spark/examples/agaricus), [Python](../../examples/Spark-ETL+XGBoost/agaricus/python/com/nvidia/spark/examples/agaricus)

- Getting started on on-premises clusters
    - [Standalone cluster for Scala](../../docs/get-started/xgboost-examples/on-prem-cluster/standalone-scala.md)
    - [Standalone cluster for Python](../../docs/get-started/xgboost-examples/on-prem-cluster/standalone-python.md)
    - [YARN for Scala](../../docs/get-started/xgboost-examples/on-prem-cluster/yarn-scala.md)
    - [YARN for Python](../../docs/get-started/xgboost-examples/on-prem-cluster/yarn-python.md)
    - [Kubernetes](../../docs/get-started/xgboost-examples/on-prem-cluster/kubernetes-scala.md)
- Getting started on cloud service providers
    - Amazon AWS
        - [EC2](../../docs/get-started/xgboost-examples/csp/aws/ec2.md)
    - [Databricks](../../docs/get-started/xgboost-examples/csp/databricks/databricks.md)
  

> ### 2. Getting Started From Notebooks

Reviewer: ```suggestion
### 2. Getting Started by using Notebooks
```

# Spark-ETL+XGBoost examples

## 1. Prepare the dataset Spark-ETL+XGBoost examples codes

> You can get a small size datasets for each example in the [datasets](../../datasets) folder. 

Reviewer 2: ```suggestion
You can get a small size dataset for each example in the [datasets](../../datasets) folder. 
```
